src/main/java/net/sf/jabref/logic/cleanup/RenamePdfCleanup.java

```diff
@@ -83,7 +85,12 @@ public RenamePdfCleanup(boolean onlyRelativePaths, BibDatabaseContext databaseCo
             }
 
             //do rename
-            boolean renameSuccessful = FileUtil.renameFile(expandedOldFilePath, newPath);
+            boolean renameSuccessful = false;
+            try {
+                renameSuccessful = FileUtil.renameFile(Paths.get(expandedOldFilePath), Paths.get(newPath));
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
Please use/introdue logging with the Logger class and then log the excpetion with LOGGER.error("message", ex)